server/vissv2server/vissv2server.go

```diff
@@ -794,7 +794,7 @@ func main() {
 	vssJson := parser.String("", "vssJson", &argparse.Options{Required: false, Help: "path and name vssPathlist json file", Default: "../vsspathlist.json"})
 	stateDB := parser.Selector("s", "statestorage", []string{"sqlite", "redis", "apache-iotdb", "none"}, &argparse.Options{Required: false,
 		Help: "Statestorage must be either sqlite, redis, apache-iotdb, or none", Default: "redis"})
-	historySupport := parser.Flag("s", "history", &argparse.Options{Required: false, Help: "Support for historic data requests", Default: false})
+	historySupport := parser.Flag("j", "history", &argparse.Options{Required: false, Help: "Support for historic data requests", Default: false})
```

Comment:
@slawr @UlfBj 

When trying to run vissv2server I got an error, which seemed to depend that we have "s" both here and on line 795 above. Seems to be a merge regression as both lines were added/changed regularly. I chnaged to "j" here to just get something unique, maybe you can propose something better.